lib/openstax/exchange/client/client.rb

```diff
@@ -0,0 +1,67 @@
+module OpenStax
+  module Exchange
+    module Client
+
+      @use_real_client = true
+
+      def self.configure
+        yield configuration
+      end
+
+      def self.configuration
+        @configuration ||= ClientConfiguration.new
+      end
+
+      def self.reset!
+        @client = nil
+      end
+
+      def self.use_real_client
+        @use_real_client = true
+      end
+
+      def self.use_fake_client
+        @use_real_client = false
+      end
+
+      def self.use_real_client?
+        !!@use_real_client
+      end
+
+      def self.create_identifier
+        begin
+          client.create_identifier
+        rescue StandardError => error
+          raise ClientError.new("create_identifier failure", error)
+        end
+      end
+
+      def self.create_multiple_choice(*args)
+        begin
+          client.create_multiple_choice(*args)
+        rescue StandardError => error
```

Comment:
I'd try to make this new method more general by passing a block:

```rb
def rescue_with(error_message)
  begin
    yield # or specify block argument and use block.call if something fancier is needed
  rescue StandardError => error
    raise ClientError.new(error_message, error)
  end
end
```

That way you can use it for `self.client` too.